<launch>
  <node pkg="coppelia_ros2_pkg" exec="coppelia_simulator" name="coppelia_simulator" output="screen" launch-prefix="xterm -e">
    # SIMULATOR PARAMETERS
    <param name="coppelia_root_dir" value="~/CoppeliaSim_Edu_V4_3_0_rev12_Ubuntu20_04" />
    <param name="coppelia_scene_path" value="$(find-pkg-share coppelia_ros2_pkg)/scenes/ros2_mobile_robot.ttt" />
    <param name="coppelia_headless" value="False" />
    
    # SCENE PARAMETERS
    <param name="num_beacons" value="7" />
    <param name="show_laser" value="True" />
  </node>
</launch>
